diff --git a/node_modules/vaul/dist/index.js b/node_modules/vaul/dist/index.js
index 7f343be..4e4a4f1 100644
--- a/node_modules/vaul/dist/index.js
+++ b/node_modules/vaul/dist/index.js
@@ -1371,7 +1371,8 @@ function Root({ open: openProp, onOpenChange, children, onDrag: onDragProp, onRe
             }
             setIsOpen(open);
         },
-        open: isOpen
+        open: isOpen,
+        modal: modal
     }, /*#__PURE__*/ React__namespace.default.createElement(DrawerContext.Provider, {
         value: {
             activeSnapPoint,
diff --git a/node_modules/vaul/dist/index.mjs b/node_modules/vaul/dist/index.mjs
index 0dbbce5..b40bf47 100644
--- a/node_modules/vaul/dist/index.mjs
+++ b/node_modules/vaul/dist/index.mjs
@@ -1349,7 +1349,8 @@ function Root({ open: openProp, onOpenChange, children, onDrag: onDragProp, onRe
             }
             setIsOpen(open);
         },
-        open: isOpen
+        open: isOpen,
+        modal: modal
     }, /*#__PURE__*/ React__default.createElement(DrawerContext.Provider, {
         value: {
             activeSnapPoint,
